<?xml version="1.0" encoding="UTF-8"?>
<!-- Generated by org.testng.reporters.JUnitXMLReporter -->
<testsuite hostname="LAPTOP-L8O5THTG" ignored="0" name="Default test" tests="2" failures="1" timestamp="01 Mar 2023 22:56:50 GMT" time="15.846" errors="0">
  <testcase name="gettitle" time="0.019" classname="testngconcepts.TC004_orangehrmapplication"/>
  <testcase name="forgotpassword" time="1.483" classname="testngconcepts.TC004_orangehrmapplication">
    <failure type="java.lang.AssertionError" message="Reset page not opened expected [Reset Password] but found [[[ChromeDriver: chrome on WINDOWS (3ea7700c18c0585c1c7cbeb962f0106a)] -&amp;gt; xpath: //button[@type = &amp;apos;submit&amp;apos;]]]">
      <![CDATA[java.lang.AssertionError: Reset page not opened expected [Reset Password] but found [[[ChromeDriver: chrome on WINDOWS (3ea7700c18c0585c1c7cbeb962f0106a)] -> xpath: //button[@type = 'submit']]]
at org.testng.Assert.fail(Assert.java:96)
at org.testng.Assert.failNotEquals(Assert.java:776)
at org.testng.Assert.assertEqualsImpl(Assert.java:137)
at org.testng.Assert.assertEquals(Assert.java:118)
at testngconcepts.TC004_orangehrmapplication.forgotpassword(TC004_orangehrmapplication.java:63)
at sun.reflect.NativeMethodAccessorImpl.invoke0(Native Method)
at sun.reflect.NativeMethodAccessorImpl.invoke(Unknown Source)
at sun.reflect.DelegatingMethodAccessorImpl.invoke(Unknown Source)
at java.lang.reflect.Method.invoke(Unknown Source)
at org.testng.internal.MethodInvocationHelper.invokeMethod(MethodInvocationHelper.java:124)
at org.testng.internal.Invoker.invokeMethod(Invoker.java:583)
at org.testng.internal.Invoker.invokeTestMethod(Invoker.java:719)
at org.testng.internal.Invoker.invokeTestMethods(Invoker.java:989)
at org.testng.internal.TestMethodWorker.invokeTestMethods(TestMethodWorker.java:125)
at org.testng.internal.TestMethodWorker.run(TestMethodWorker.java:109)
at org.testng.TestRunner.privateRun(TestRunner.java:648)
at org.testng.TestRunner.run(TestRunner.java:505)
at org.testng.SuiteRunner.runTest(SuiteRunner.java:455)
at org.testng.SuiteRunner.runSequentially(SuiteRunner.java:450)
at org.testng.SuiteRunner.privateRun(SuiteRunner.java:415)
at org.testng.SuiteRunner.run(SuiteRunner.java:364)
at org.testng.SuiteRunnerWorker.runSuite(SuiteRunnerWorker.java:52)
at org.testng.SuiteRunnerWorker.run(SuiteRunnerWorker.java:84)
at org.testng.TestNG.runSuitesSequentially(TestNG.java:1208)
at org.testng.TestNG.runSuitesLocally(TestNG.java:1137)
at org.testng.TestNG.runSuites(TestNG.java:1049)
at org.testng.TestNG.run(TestNG.java:1017)
at org.testng.remote.AbstractRemoteTestNG.run(AbstractRemoteTestNG.java:115)
at org.testng.remote.RemoteTestNG.initAndRun(RemoteTestNG.java:251)
at org.testng.remote.RemoteTestNG.main(RemoteTestNG.java:77)
]]>
    </failure>
  </testcase> <!-- forgotpassword -->
</testsuite> <!-- Default test -->
